Ladybird/WebContentView.h

```diff
@@ -51,6 +51,7 @@ class WebContentView final
     virtual ~WebContentView() override;
 
     Function<String(Web::HTML::ActivateTab)> on_new_tab;
+    Function<String(const AK::URL&, Web::HTML::ActivateTab)> on_open_url_in_new_tab;
```

Comment:
We already have `on_link_click`, please don't deviate from `Browser` when possible.

https://github.com/SerenityOS/serenity/blob/c048cf2004c9cda9ed6533b81b586600b740c14b/Userland/Applications/Browser/Tab.cpp#L265-L271

Reply:
I'd need to check what to do, it seems Ladybird already deviates a little from userland browser.

Reply:
Please check now, the code looks a little more like serenityos's Browser but with normal current browsers' activate behavior, ie: middle click anything opens in an onfocused tab

Reply:
Thanks!